sdk/monitor/Azure.ResourceManager.Monitor/tests/TestCase/MetricAlertCollectionTests.cs

```diff
@@ -14,10 +15,22 @@ namespace Azure.ResourceManager.Monitor.Tests
     public class MetricAlertCollectionTests : MonitorTestBase
     {
         public MetricAlertCollectionTests(bool isAsync)
-           : base(isAsync)
+           : base(isAsync)//, RecordedTestMode.Record)
         {
         }
 
+        #region storage account
+        public async Task<string> GetStorageAccountId()
+        {
+            var resourceGroup = await CreateResourceGroupAsync();
+            var storageCeollection = resourceGroup.GetStorageAccounts();
+            string accountName = Recording.GenerateAssetName("metrictests");
+            var storageContent = ResourceDataHelper.GetContent();
+            var storageAccount = await storageCeollection.CreateOrUpdateAsync(WaitUntil.Completed, accountName, storageContent);
+            return storageAccount.Value.Data.Id;
+        }
+        #endregion
```

Comment:
We kind of want to exclude these operations from the recording. Otherwise whenever storage PR has an api-version upgrade, we will have to record monitor's test cases again as well

Reply:
You could refer to this on how to do that: https://github.com/Azure/azure-sdk-for-net/blob/abf92e3b4d559ac1cee311a274b7a3e3f662248c/sdk/servicebus/Azure.ResourceManager.ServiceBus/tests/Tests/ServiceBusNamespaceTests.cs#L566
Wrap the content of this function into a `if {} else {}`, if it is in playback mode, we just construct an id of storage account and return, otherwise we stop recording and call the API to create the storage account and return the id

Reply:
fixed